### PR TITLE
Add normalizers constructors with max aggregator and normalizer

### PR DESCRIPTION
**Pull request by AlexVanin**
Now both normalizers and aggregators have public constructors for
external usage. We also added max aggregator and max normalizer because
it is commonly used in calculations.

### CODE REVIEW

File: aggregator_test.go [hunk 1]
```diff
@@ -124,23 +128,38 @@ func TestSigmoidNorm_Normalize(t *testing.T) {
...
+
+	t.Run("max norm should equal 1 at max value", func(t *testing.T) {
+		norm := NewMaxNorm(10)
 		require.InEpsilon(t, 1.0, norm.Normalize(10), eps)
 	})
 }
 
 func TestBucket_TraverseTree(t *testing.T) {
 	var (
-		meanAF = AggregatorFactory{New: func() Aggregator { return new(meanAgg) }}
```
Comment (fyrchik) [inline aggregator_test.go]: There is no need to have an additional call here. It is ok not to use public constructors in internal implementation.

Comment (AlexVanin) [reply aggregator_test.go]: In the test I wanted to cover publicly available functions as more as I can.

Comment (fyrchik) [reply aggregator_test.go]: Could you then move this changes into a separate commit?

Comment (fyrchik) [reply aggregator_test.go]: Otherwise, LGTM.

**Pull request merged**